pkg/backend/create.go

```diff
@@ -67,6 +67,13 @@ func (c *CreateCommand) Exec(in io.Reader, out io.Writer) error {
 	c.Input.UseSSL = fastly.Compatibool(c.UseSSL)
 	c.Input.SSLCheckCert = fastly.Compatibool(c.SSLCheckCert)
 
+	if c.UseSSL && c.Input.Port == 0 {
+		if c.Globals.Flag.Verbose {
+			text.Warning(out, "SSL to the backend was set but no port was specified, so default port :443 will be used")
```

Comment:
"SSL to the backend" sounds strange to me personally and I think its better to use the explicit name of the flag when you're saying it was "set". Ports don't have `:`.

```suggestion
			text.Warning(out, "Use-ssl was set but no port was specified, so default port 443 will be used")
```